kivy/data/style.kv

```diff
@@ -538,6 +538,10 @@
         BoxLayout:
             id: container
 
+
+<Carousel>:
+    _prev_equals_next: self.loop and (len(self.slides) == 2)
```

Comment:
I like using KV like this because it's much simpler compared to doing the same thing in python. But if you don't think so, I don't do this anymore on this project.

Regardless of that, I'll remove that part because I'll make `_prev_equals_next` into non KivyProperty as you suggested.